tor/validation/formatting_validation.py

```diff
@@ -246,13 +246,14 @@ def check_for_invalid_header(transcription: str) -> Optional[FormattingIssue]:
     Valid: *Video Transcription: Test*
     Valid: *Image Transcription*
     Invalid: *Random Transcription*
+    Invalid: Test *Image Transcription*
     """
     header = transcription.split("---")[0]
     return (
         FormattingIssue.INVALID_HEADER
         if not any(
             [
-                re.search(r"\*{}.*\*".format(i), header) is not None
+                re.search(r"^\**{}.*\*".format(i), header) is not None

```

Comment:
I think we should put a `\s*` after the `^` so that the user can put some whitespace before the header without triggering this check.

I'm also not sure about the `\**` part. That also accepts no asterisk at all or as many as the user wants. I assume this is because of the bold header check, but if anything we should allow an extra one there, not zero or more.

I think in the future I will rewrite the header check completely to give us more control of what we check and which message we display to the user.

Reply:
Whoops...it was meant to be a `+`

Yeah, the header check is a bit of a mess right now, especially because of the bold header check. Likely better to just merge the two at some point. Either way, resolving this now.

Reply:
This should be resolved as of the latest commit.